scripts/deploy_bug.py

```diff
@@ -108,9 +108,10 @@ def main(argv):
         if len(commit['parents']) > 1:
             continue
 
-        print('%s: %s' % (
+        print('%s: %s (%s)' % (
             commit['sha'][:7],
-            commit['commit']['message'].splitlines()[0][:80]
+            commit['commit']['message'].splitlines()[0][:80],
+            (commit['author'] or {}).get('login', '?')
```

Comment:
Our of pure curiosity, why this instead of `commit.get('author', {})`?

Reply:
I copied this from my `status.py` script which I wrote a while ago. I think the problem is that we need to account for when the value for `commit['author']` is `None` as well as a dict that might not have a `login` key.

So the first part:

```
commit['author'] or {}
```

handles the `None` case because that's falsey and makes sure the result is a dict that has a `.get()` method.

I can make this better via a comment or something if you think that's worthwhile.

Reply:
Scratch that--I'll make it better.